test/season_stat_coach_test.rb

```diff
@@ -27,13 +22,39 @@ def test_it_can_get_game_teams_by_season
 
   def test_it_can_get_coaches_by_season
     assert_instance_of Array, @season_stat_coach.coaches_by_season("20122013")
+    assert_equal "John Tortorella", @season_stat_coach.coaches_by_season("20122013")[0]
     assert_equal "Claude Julien", @season_stat_coach.coaches_by_season("20122013")[1]
+    assert_equal "Dan Bylsma", @season_stat_coach.coaches_by_season("20122013")[2]
+    assert_equal "Mike Babcock", @season_stat_coach.coaches_by_season("20122013")[3]
+    assert_equal "Joel Quenneville", @season_stat_coach.coaches_by_season("20122013")[4]
+    assert_equal "Paul MacLean", @season_stat_coach.coaches_by_season("20122013")[5]
+    assert_equal "Michel Therrien", @season_stat_coach.coaches_by_season("20122013")[6]
+    assert_equal "Mike Yeo", @season_stat_coach.coaches_by_season("20122013")[7]
+    assert_equal "Darryl Sutter", @season_stat_coach.coaches_by_season("20122013")[8]
+    assert_equal "Ken Hitchcock", @season_stat_coach.coaches_by_season("20122013")[9]
+    assert_equal "Bruce Boudreau", @season_stat_coach.coaches_by_season("20122013")[10]
+    assert_equal "Jack Capuano", @season_stat_coach.coaches_by_season("20122013")[11]
+    assert_equal "Adam Oates", @season_stat_coach.coaches_by_season("20122013")[12]
+    assert_equal "Todd Richards", @season_stat_coach.coaches_by_season("20122013")[13]
     assert_equal 14, @season_stat_coach.coaches_by_season("20122013").length
     assert_nil @season_stat_coach.coaches_by_season("20122013")[29]
```

Comment:
just wanted to be a little more robust here and check to see it was working for all coaches, since there aren't that many